src/app/login/login-form/login-form.component.scss

```
@@ -188,25 +162,30 @@
 .login-progress {
   height: 3px;
   border-radius: 4px;
-  margin: -0.5rem 0 0.5rem;
+  margin: -0.25rem 0 0.5rem;
 
   ::ng-deep {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)